CRM/Core/Form/RecurringEntity.php

```diff
@@ -392,7 +393,7 @@ public static function postProcess($params, $type, $linkedEntities = []) {
           }
         }
 
-        //Set type for API
+        // FIXME: This is the worst way possible to convert a table name to an api entity name
         $apiEntityType = explode("_", $type);
         if (!empty($apiEntityType[1])) {
           $apiType = $apiEntityType[1];
```

Comment:
Honestly nearly every line of code in this file had me gnashing my teeth. Since this is a hotfix PR I resisted the urge to do further cleanup but couldn't help but add this comment.

Reply:
Thanks. I see what you mean.

`if ($result['error'])`

Reply:
@colemanw just to be clear - you say this is a hot-fix PR - but I'm under the understanding that it IS a master-only regression - is that correct? (just checking because hot-fix might imply you meant to target the rc)

Reply:
It's master-only.

Reply:
Yea, master only. So... warm fix? Luke-warm? Tepid?

Reply:
gnashing teeth => hot-under-the-collar-fix ?

Reply:
I couldn't resist any longer, this will prevent me grinding my teeth down quite so fast: #27328